pkg/insights/resyncer.go

```diff
@@ -399,8 +437,20 @@ func (r *resyncer) createOrUpdateMeshInsight(mesh string) error {
 		}
 	}
 
-	err := manager.Upsert(r.rm, model.ResourceKey{Mesh: model.NoMesh, Name: mesh}, core_mesh.NewMeshInsightResource(), func(resource model.Resource) error {
-		insight.LastSync = proto.MustTimestampProto(core.Now())
+	hash, err := proto.Hash(insight)
```

Comment:
I wonder whether you need the hash? We fetch the latest copy of the mesh insight in `meedResyncMeshInsight`, so can we just compare the new and old protobufs for equality?

Reply:
`needResyncServiceInsight` is only executed from ticker, not from events. Actually, there is no point in retrieving Insights in `needX` methods anymore.

But that is a good point about hash itself. I wanted to avoid keeping insight in memory, but it doesn't make sense since insight is small enough. I removed the hash.

Thanks!